build/yamls/base/controller-rbac.yml

```diff
@@ -83,3 +83,19 @@ subjects:
   - kind: ServiceAccount
     name: antrea-controller
     namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: antrea-controller-antctl
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  # This is to allow antctl to be executed inside the Antrea Controller Pod and
+  # authenticate with Controller API server using the Controller's
+  # ServiceAccount token.
+  name: antctl
+subjects:
+  - kind: ServiceAccount
+    name: antrea-controller
+    namespace: kube-system
```

Comment:
Just to confirm you prefer to create a new ClusterRoleBinding to associate antrea-controller account and antctl role separately, than adding another subject to the existing ClusterRoleBinding antctl?

Reply:
Humm.. I do not know which way is better. What is your suggestion?

Reply:
I prefer adding a subject to the existing rolebinding, as I think role is already an abstract of permissions, the rolebinding is more focused on the role and describes who get the role, maybe no need to have one object per subject. Like the built-in rolebinding `system::extension-apiserver-authentication-reader`, it's to describe kube-controller-manager and kube-scheduler have extension-apiserver-authentication-reader role.
```
  roleRef:
    apiGroup: rbac.authorization.k8s.io
    kind: Role
    name: extension-apiserver-authentication-reader
  subjects:
  - apiGroup: rbac.authorization.k8s.io
    kind: User
    name: system:kube-controller-manager
  - apiGroup: rbac.authorization.k8s.io
    kind: User
    name: system:kube-scheduler
```

Reply:
Ok. Changed to add a subject for antctl role binding. Only thing is I feel easier to track if we put all role binding for controller in controller-rbac.yml.

Reply:
Yes, I thought it too, but thinking we would not track who get the antctl role from another perspective, so thought one side has to be sacrificed.